src/main/java/org/vietabroader/view/MainView.java

```diff
@@ -130,30 +133,32 @@ private JPanel createWorkspacePanel() {
         c.gridx = 1;
         c.gridy = 0;
         c.weightx = 1;
-        c.fill = GridBagConstraints.HORIZONTAL;
+        c.gridwidth = 3;
+        c.fill = GridBagConstraints.BOTH;
```

Comment:
The value of c.fill is set to BOTH here so it is still BOTH for the rest of the method unless you change it to something else. That's why the label "to" fills the whole cell, so setting the anchor does not have any effect at all.